.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow will publish generator-symphony to npmjs registry
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Publish
+on:
+  release:
+    types: [created]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
how does it know which tag/branch to checkout? automatically from the event?

Reply:
I'd say so, it should get the branch to checkout directly from the release event. The release workflow in bdk java/python are both done like this.

Reply:
I confirm